app/models/studio.rb

```diff
@@ -1,3 +1,15 @@
 class Studio < ApplicationRecord
   has_many :movies
+
+  def list_movies_details
+    movies_details = movies.pluck(:title, :creation_year, :genre)
+
+    movies_details = movies_details.flat_map do |movie|
+      movie.join(", ")
```

Comment:
This is more of a "formatting" type method, which is not necessarily the purview of the Model - maybe a helper method in a module, or even a private method in a model or controller. But as such, there's nothing really testable about this method, so while it is cool, it's a bit overkill for a model since it's more about view formatting than retrieving a specific collection from your database.